### DISCUSSION TITLE
Excuse me, do you have the circuit diagram of DSO-500K?

### DISCUSSION THREAD

**Discussion by ht98006** [fhdm-dev/scoppy#99] [Q&A]
Excuse me, do you have the circuit diagram of DSO-500K?i want to do a test

**Comment by fhdm-dev** [comment]
Hi,

The Schematic & PCB design are available at [OSHWLAB](https://oshwlab.com/fruitloop57/fscope-250k5-v2_copy_copy_copy_copy)
